### PR TITLE
Add information in README about discontinuation of project

### PR DESCRIPTION
**Pull request by szadam**
Preview: https://github.com/szadam/vmem/blob/master/README.md

### CODE REVIEW

**Comment by kilobyte** [comment]
Looks good.

File: README.md [hunk 1]
```diff
@@ -6,6 +6,17 @@
 [![Release version](https://img.shields.io/github/release/pmem/vmem.svg?sort=semver)](https://github.com/pmem/vmem/releases/latest)
 [![Coverage Status](https://codecov.io/github/pmem/vmem/coverage.svg?branch=master)](https://codecov.io/gh/pmem/vmem/branch/master)
 
+## ⚠️ Discontinuation of the project
+The **ibvmem** and **libvmmalloc** project will no longer be maintained by Intel.
```
Comment (lukaszstolarczuk) [inline README.md]: 'libvmem' (misspell)

Comment (lukaszstolarczuk) [inline README.md]: hmm, actually `libvmem and libvmmalloc project` is an unfortunate sentence 😉 perhaps we could say:

1. libvmem project will no...
or
2. libvmem and libvmmalloc libraries will no...

**Review by lukaszstolarczuk — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (szadam) [reply README.md]: done

Comment (szadam) [reply README.md]: done

**Review by lukaszstolarczuk — APPROVED** [review]

**Pull request merged**